README.ko.md

```diff
@@ -0,0 +1,89 @@
+# caver-js-examples
+**caver-js-examples** 프로젝트는 SDK를 사용하는 분들이 좀 더 쉽고 빠르게 SDK를 사용하실 수 있게끔 도움을 드리기 위한 프로젝트입니다.
+다양한 상황에 맞는 예제코드를 직접 확인하시고 실행해보실 수 있습니다.
+
+## 1. 선행되어야 하는 작업
+### 1.1 실습을 위해 필요한 소프트웨어 설치
+> 본 프로젝트를 구성할 때 사용한 소프트웨어들을 기술합니다. 원활한 실습 진행을 위해 아래 기술한 소프트웨어들은 실습 전에 설치 해주시기 바랍니다.
+* [Node.js](https://nodejs.org/en/download/) (아래 버전 중 하나를 권장드립니다.)
+    * lts/erbium ([12.21.0](https://nodejs.org/dist/latest-v12.x/))
+    * lts/fermium ([14.16.0](https://nodejs.org/dist/latest-v14.x/))
+* [npm](https://www.npmjs.com/get-npm)
+* [gcc-c++](https://gcc.gnu.org/)
+
+### 1.2 KAS 회원가입
+Caver SDK는 [Klaytn EndpointNode(이하 Klaytn EN)](https://docs.klaytn.com/node/endpoint-node)와 함께 사용할 수 있도록 만들어져 있습니다.
+[EN](https://docs.klaytn.com/node/endpoint-node)을 직접 운영하지 않아도 본 예제코드들을 테스트해볼 수 있도록, caver-js-examples 프로젝트는 [KAS](https://klaytnapi.com)의 [NODE API](https://refs.klaytnapi.com/en/node/latest)를 사용합니다.
+
+KAS를 사용하기 위해서는 [KAS](https://klaytnapi.com)에 가입하셔야 합니다. 가입하시면 예제코드 실행에 필요한 `access key id`와 `secret access key`를 발급받으실 수 있습니다.
+발급받은 `access key id`와 `secret access key`는 [2. 사용법](https://github.com/klaytn/caver-js-examples/blob/master/README.ko.md#2-%EC%82%AC%EC%9A%A9%EB%B2%95) 섹션을 참고하셔서 적절한 위치에 값을 넣어주신 후 사용하시면 됩니다.
+
+### 1.3 계정 준비
+5 KLAY 이상을 보유하고 있는 테스트 용도의 계정을 5개 이상 준비해주시기 바랍니다.
```

Comment:
Can you explain why user has to create at least 5 accounts?

Reply:
Actually I think three would be enough.
The reason I said 5 is because of the use case below.
1. Fee Payer Account
2. Deployer Account (It doesn't matter if the Sender and Deployer accounts are the same.)
3. Sender Account
4. Sender Account to execute [account/update_account_with_account_key_weighted_multisig](https://github.com/klaytn/caver-js-examples/blob/master/account/update_account_with_account_key_weighted_multisig/boilerplate.js) scenario. When a user updates an account, that account is difficult to use for another scenarios.
5. Sender Account to execute [account/update_account_with_account_key_role_based](https://github.com/klaytn/caver-js-examples/blob/master/account/update_account_with_account_key_role_based/boilerplate.js) scenario. When a user updates an account, that account is difficult to use for another scenarios.

It's also based on my experience. When I ran the examples, I felt that 5 was enough.